app/admin/user.rb

```diff
@@ -3,6 +3,13 @@
 ActiveAdmin.register User do
   permit_params :email, :first_name, :last_name, :username, :password, :password_confirmation
 
+  member_action :impersonate, method: :post do
```

Comment:
Should we also check for `ENV['IMPERSONATION_URL']` before defining this action?